src/mscorlib/model.xml

```diff
@@ -3693,21 +3733,26 @@
       <Member Name="#ctor" />
       <Member Name="get_CultureInfo" />
       <Member Name="get_CultureName" />
+      <Member Name="get_CodeBase" />
```

Comment:
So far I added the APIs listed in the tracking spreadsheet. I'm diffing S.P.CoreLib against mscorlib ref to find the missing members and I'm seeing some are still not exposed. I will expose the rest as well.